src/inmanta/protocol/rest/__init__.py

```diff
@@ -469,21 +474,104 @@ async def process_return(self, config: common.UrlMethod, result: Apireturn) -> c
 
             else:
                 raise exceptions.ServerError(
-                    f"Method {config.method_name} returned an invalid result {result} instead of a status code or tupple"
+                    f"Method {self._config.method_name} returned an invalid result {result} instead of a status code or tupple"
                 )
 
             if body is not None:
-                if config.properties.reply:
+                if self._config.properties.reply:
                     return common.Response.create(
                         ReturnValue(status_code=code, response=body),
-                        config.properties.envelope,
-                        config.properties.envelope_key,
+                        self._config.properties.envelope,
+                        self._config.properties.envelope_key,
                     )
                 else:
                     LOGGER.warning("Method %s returned a result although it has no reply!")
 
             return common.Response.create(ReturnValue(status_code=code, response=None), envelope=False)
 
+    def _parse_and_validate_auth_token(self) -> None:
+        """Get the auth token provided by the caller and decode it.
+
+        :return: A mapping of claims
+        """
+        header_value: Optional[str] = None
+        header_name: Optional[str] = None
+        cfg_name: Optional[str] = None
+        for name in auth.AuthJWTConfig.list():
```

Comment:
I would configure the headers outside the specific JWT config, as they are not related here. (i.e. any header is for any JWT issues)

Reply:
done